From ad8e00d11abbcec1f220615f927eba3c8066c3c6 Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexdeucher@gmail.com>
Date: Thu, 10 Aug 2017 14:22:21 -0400
Subject: [PATCH 3033/5855] drm/amd/display/dc: Make dce110_validate_bandwidth
 static (v2)

Not used outside of the dce110_resource.c.

v2: make a bunch of other functions static as well.

Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Reviewed-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c b/drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c
index c677f98..a7f30dc 100644
--- a/drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c
+++ b/drivers/gpu/drm/amd/display/dc/dce110/dce110_resource.c
@@ -565,7 +565,7 @@ static const struct encoder_feature_support link_enc_feature = {
 		.flags.bits.IS_YCBCR_CAPABLE = true
 };
 
-struct link_encoder *dce110_link_encoder_create(
+static struct link_encoder *dce110_link_encoder_create(
 	const struct encoder_init_data *enc_init_data)
 {
 	struct dce110_link_encoder *enc110 =
@@ -813,7 +813,7 @@ static enum dc_status build_mapped_resource(
 	return DC_OK;
 }
 
-bool dce110_validate_bandwidth(
+static bool dce110_validate_bandwidth(
 	const struct core_dc *dc,
 	struct validate_context *context)
 {
@@ -927,7 +927,7 @@ static bool dce110_validate_surface_sets(
 	return true;
 }
 
-enum dc_status dce110_validate_with_context(
+static enum dc_status dce110_validate_with_context(
 		const struct core_dc *dc,
 		const struct dc_validation_set set[],
 		int set_count,
@@ -971,7 +971,7 @@ enum dc_status dce110_validate_with_context(
 	return result;
 }
 
-enum dc_status dce110_validate_guaranteed(
+static enum dc_status dce110_validate_guaranteed(
 		const struct core_dc *dc,
 		struct dc_stream_state *dc_stream,
 		struct validate_context *context)
-- 
2.7.4

